spec/spec_helper.rb

```diff
@@ -1,22 +1,2 @@
-require 'rubygems'
-require "codeclimate-test-reporter"
-CodeClimate::TestReporter.start
-
-# COVERAGE=true rspec
-if ENV['COVERAGE']
-  require 'simplecov'
-  SimpleCov.start
-end
-
+$LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.